bin/vint

```diff
@@ -17,9 +17,9 @@ from pathlib import Path
 #   $ python -m unittest discover test
 
 if __name__ == '__main__':
-    vint_dir = Path(__file__).resolve().parent.parent
+    vint_root = Path(__file__).resolve().parent.parent
+    sys.path.append(str(vint_root))
 
-    sys.path.append(str(vint_dir))
     import vint
 
-    vint.main()
+    sys.exit(vint.main())
```

Comment:
`sys.exit()` gracefully exits from Python and whatever needed cleanups are performed.
`vint.main()` returns `None` on success, which translates into `0` success code.
See https://docs.python.org/3/library/sys.html#sys.exit